src/SSVOpenHexagon/Core/MenuGame.cpp

```diff
@@ -402,6 +421,8 @@ void MenuGame::upAction()
 }
 void MenuGame::downAction()
 {
+    if(noActions) return;
```

Comment:
Again, this is not immediately clear. Maybe rename `noActions` to something more specific? Maybe something like `suppressActionsDueToDialogBox`?

Reply:
Also, sorry... but I like my braces :)

```suggestion
    if(noActions) { return; }
```